client/components/first-view/index.jsx

```diff
@@ -100,14 +100,21 @@ const FirstView = React.createClass( {
 	},
 
 	updateDocumentStylesForVisibleFirstView() {
-		document.documentElement.classList.add( 'no-scroll', 'is-first-view-visible', 'is-first-view-active' );
+		document.documentElement.classList.add( 'no-scroll', 'is-first-view-active' );
```

Comment:
Why do we need to manually set classes here? Can we instead set ui attributes on state and print them on `layout`?